api/fasit_test.go

```diff
@@ -279,7 +279,7 @@ func TestGetLoadBalancerConfig(t *testing.T) {
 		resource, err := fasit.getLoadBalancerConfig("application", "environment")
 
 		assert.NoError(t, err)
-		assert.Equal(t, 2, len(resource.ingresses))
+		assert.Equal(t, 5, len(resource.ingresses))
```

Comment:
I know this isnt you, but there should be a test that covers 0 ingresses and 1 ingress as well as 5